### PR TITLE
login

### PR DESCRIPTION
**Pull request by aryehlev**
this is a login that connects to dtabase- also dont see need for routes folder and routing.py, i worked straight from routes.py

### CODE REVIEW

File: .github/workflows/pylint.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: Pylint
```
Comment (simzacks) [inline .github/workflows/pylint.yml:1]: looks good. But do you know how to add this as its own PR, so it isn't connected to the login PR?
See: https://stackoverflow.com/questions/8450036/how-to-open-multiple-pull-requests-on-github

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "C:\\Users\\97254\\AppData\\Local\\Microsoft\\WindowsApps\\python.exe"
+}
```
Comment (simzacks) [inline .vscode/settings.json:3]: this file shouldn't be in the repo.
You can exclude it by adding the .vscode folder to the .gitignore file

File: bootstrap.sh [hunk 3]
```diff
@@ -2,10 +2,21 @@
 
 apt-get update
 apt-get install python3 python3-pip -y
-pip3 install flask Flask-SQLAlchemy SQLAlchemy pymysql
+pip3 install -r requirments.txt
+# pip3 install flask Flask-SQLAlchemy SQLAlchemy pymysql Flask-WTF Flask-Bootstrap Flask_login pytest
```
Comment (simzacks) [inline bootstrap.sh:6]: please remove the comment.
comments should be here to explain code, not for code you decided to erase

File: bootstrap.sh [hunk 4]
```diff
@@ -2,10 +2,21 @@
...
+
+
+
+
+
+export FLASK_APP=/vagrant/app.py
+
+export FLASK_ENV=development
+
+flask run -h 0.0.0.0 -p 5000
```
Comment (simzacks) [inline bootstrap.sh:22]: this code is in here twice

File: superx/app.py [hunk 5]
```diff
@@ -1,13 +1,24 @@
...
+SQLALCHEMY_ECHO = False
+SQLALCHEMY_TRACK_MODIFICATIONS = False
+
+db = SQLAlchemy(app)
 db.init_app(app)
 
 from routing import *
 
-app.run(debug=True)
+# app.run(debug=True)
```
Comment (simzacks) [inline superx/app.py:24]: remove comment please

File: superx/models/__init__.py [hunk 6]
```diff
@@ -1,14 +1,22 @@
...
-
+from flask_login import LoginManager, UserMixin
 from app import db
 
 
-class User(db.Model):
+class User(UserMixin, db.Model):
     __tablename__ = 'user'
 
     id = Column(Integer, primary_key=True)
```
Comment (simzacks) [inline superx/models/__init__.py:9]: is this an auto_increment or are you planning on typing in the id?

File: superx/models/__init__.py [hunk 7]
```diff
@@ -1,14 +1,22 @@
...
-    name = Column(Text)
-    password = Column(Text)
+    name = Column(db.String(15))
+    email = Column(db.String(50), unique=True)
+    password = Column(db.String(80))
+
+class test(db.Model):
```
Comment (simzacks) [inline superx/models/__init__.py:14]: python name standards for classes is CamelCase, so here it would be Test

File: test/test_basic.py [hunk 8]
```diff
@@ -0,0 +1 @@
+from app import
```
Comment (simzacks) [inline test/test_basic.py:1]: This file is not ready for merge

**Review by simzacks — CHANGES_REQUESTED** [review]

File: .github/workflows/pylint.yml (hunk 1 above)
Comment (aryehlev) [reply .github/workflows/pylint.yml:1]: noted, i will put the fixes in a new PR to make sure i understand

File: superx/models/__init__.py (hunk 6 above)
Comment (aryehlev) [reply superx/models/__init__.py:9]: it should be auto_increment automatically no?

File: .github/workflows/pylint.yml (hunk 1 above)
Comment (aryehlev) [reply .github/workflows/pylint.yml:1]: oh, just realized did mistake. i will push the whole changes in new pull request. sorry :(

File: superx/models/__init__.py (hunk 6 above)
Comment (simzacks) [reply superx/models/__init__.py:9]: Probably,

**Pull request merged**